rest-service/manager_rest/dsl_functions.py

```diff
@@ -32,86 +32,113 @@
 SecretType = namedtuple('Secret', 'key value')
 
 
+def evaluate_node(node):
+    node['name'] = node['id']
+    deployment_id = node['deployment_id']
+    sm = get_storage_manager()
+    sm.get(Deployment, deployment_id, include=['id'])
```

Comment:
Why is this needed? for validating the deployment_id is for a "live" deployment?

Reply:
yes i think that is why (this isn't a new line, just copied)